### ISSUE TITLE
Should abstract away notions of "band" and "subband" from users

### ISSUE THREAD

**Issue by swh76** [slaclab/pysmurf#64]
Users should be able to target specific intervals by actual frequency, and not have to care about the SMuRF internal concepts of "bands" and "subbands."  We may also want to add an abstraction layer between users and the SMuRF DSP channel ordering, as is done with subbands.

**Comment by eyyoung24** [comment]
Is this different? https://github.com/slaclab/pysmurf/issues/5

**Comment by swh76** [comment]
Nope!  Thanks!  I'll close the other one since this one includes the subband and possible channel order.

**Comment by ruck314** [comment]
Revisiting this 2019 enhancement for housekeeping. My read of the current `main` is that the realistic user-facing piece of this request has landed, and the broader "abstract away band/subband/channel" refactor has not been pursued and is unlikely to be in its original sweeping form. I'd suggest closing.

**What's already in place**

- `SmurfTune.find_freq()` accepts `start_freq` / `stop_freq` in MHz (relative to band center); the legacy `subband` argument is documented as deprecated. See `python/pysmurf/client/tune/smurf_tune.py` (the `find_freq` definition and its docstring).
- Frequency ↔ index conversion helpers users can call directly:
  - `SmurfUtilMixin.freq_to_subband(band, freq)` in `python/pysmurf/client/util/smurf_util.py`
  - `SmurfUtilMixin.channel_to_freq(band, channel)` in `python/pysmurf/client/util/smurf_util.py`
  - `SmurfTuneMixin.freq_to_band(frequency, band_center_list)` in `python/pysmurf/client/tune/smurf_tune.py`
  - `SmurfTuneMixin.get_closest_subband(f, band)` in `python/pysmurf/client/tune/smurf_tune.py`

**What's not done (and probably shouldn't be)**

- No absolute-frequency wrappers around `tune_band`, `eta_scan_*`, `relock`, `tracking_setup`, etc. — they still take a `band` index.
- No DSP-channel abstraction over the `(band, channel)` tuple.

**Why I'd close rather than keep open**

The SMuRF stack is band-organized at the hardware level — the band index selects an AMC tile / DAC / ADC pair, not just a numerical bucket. Real tuning workflows are already per-band (different RF paths, different resonator populations, different attenuators). After 6+ years with no concrete design surfacing, the partial frequency-first improvements above seem to have absorbed the practical user need.

If a specific frequency-first convenience is still wanted (e.g., a `find_freq_at(freq_mhz, span_mhz)` that auto-selects the band), that's narrow enough to track as its own ticket against a concrete signature. I don't think the umbrella request as written here will produce a useful PR.

Recommend: close as `wontfix`, file a narrower follow-up only if there's a specific call site that wants the frequency-first wrapper.
